### DISCUSSION TITLE
Anfänger sucht Ratschlag

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#522] [General]
Hallo Leute,

Mein Nachbar und Ich möchten gern je ein OpendDTU-OnBattery System aufbauen.
Unsere Vorstellungen wären:
```
2x Astronergy Black Frame 425Wp (ggf ein 3. in Zukunft)
5kW Pylontech Speicher für den Nachbarn (ich kann gratis auf gute gebrauchte Pb Akkus zugreifen)
```
Aktuell vorhanden ist:
```
Shelly 3em
Homeassistant auf Raspberry Pi 4
```
Als MPPT Laderegler würden wir etwas von Victron nutzen wollen.

Mit ESP32 habe ich schon eine Weile andere Dinge realisiert, also Vorkenntnisse vorhanden.

Die Frage ist, was benötigen wir noch um das System fertig aufzubauen?
Gibt es irgendwelche Probleme mit den Pylontech Kassetten?

Ich Hoffe jemand kann uns beraten 

LG aus Sachsen

**Comment by genmad** [comment]
Siehe hier: https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Builds-%26-Examples für Beispiele als Ausgangspunkt.

Weiterhin kann ich nur sagen das ihr nur ähnliches Zeug machen könnt wenn ihr das gleiche Batteriespannungsniveau habt. Bei der Pylontech gehe ich davon aus, dass die US5000 gemeint ist, die hat 48V als Spannungsniveau. Deine PB Akkus können je nach Verschaltung bei typischerweise 12V, 24V, oder 48V betrieben werden. Darin werden sich eure Systeme schon unterscheiden, und daher sollten dann separate Fragen für die entsprechende Konfiguration gestellt werden.

Weiterhin halte ich 2 * 425 Wp Module an einer 5 kWh Batterie ( bei täglicher Nutzung der Batterie und nicht das Aufladen über Tage hinweg und dann den Strom nur z.B. am Wochenende zu nutzen) für zu wenig oder die Batterie für zu groß ( teuer ) für den Aufbau. 
 
Stelle bitte mal deine /eure Erwartungen und Annahmen dar, sowie eure erwarteten Verbräuche dar.
